src/ostree/ot-builtin-checkout.c

```diff
@@ -193,6 +195,7 @@ process_one_checkout (OstreeRepo           *repo,
       checkout_options.force_copy = opt_force_copy;
       checkout_options.force_copy_zerosized = opt_force_copy_zerosized;
       checkout_options.bareuseronly_dirs = opt_bareuseronly_dirs;
+      checkout_options.process_passthrough_whiteouts = opt_process_passthrough_whiteouts;
```

Comment:
I'm checking why this option is not getting down to the callee down the line in my local testing.

Reply:
```
+ ./ostree checkout --process-passthrough-whiteouts -v --repo=./testrepo-bare test/stable/x86_64 ./rootfs-out
.ostree-wh.whiteout is_overlayfs_whiteout: 1, enabled:0
not-a-whiteout-just-empty is_overlayfs_whiteout: 0, enabled:0
.ostree-wh.whiteout is_overlayfs_whiteout: 1, enabled:0
.ostree-wh.whiteout2 is_overlayfs_whiteout: 1, enabled:0
+ ls -la rootfs-out
total 8
drwxr-xr-x. 1 majopela majopela  102 Jan  1  1970 .
drwxr-xr-x. 1 majopela majopela 2058 Sep 20 14:24 ..
drwxr-xr-x. 1 majopela majopela   78 Jan  1  1970 another
-rw-r--r--. 1 majopela majopela    0 Sep 20 14:24 not-a-whiteout-just-empty
-rw-r--r--. 1 majopela majopela    0 Sep 20 14:24 .ostree-wh.whiteout
+ ls -la rootfs-out/another
total 0
drwxr-xr-x. 1 majopela majopela  78 Jan  1  1970 .
drwxr-xr-x. 1 majopela majopela 102 Jan  1  1970 ..
-rw-r--r--. 1 majopela majopela   0 Sep 20 14:24 .ostree-wh.whiteout
-rw-r--r--. 1 majopela majopela   0 Sep 20 14:24 .ostree-wh.whiteout2
+ getfattr -d -m '.*' rootfs-out/whiteout
getfattr: rootfs-out/whiteout: No such file or directory

```

I'm missing something here... ':D

Reply:
I separated those for debugging:  
```
  const gboolean is_overlayfs_whiteout = (!is_symlink && g_str_has_prefix (destination_name, OVERLAYFS_WHITEOUT_PREFIX));

  printf("%s is_overlayfs_whiteout: %d, enabled:%d\n", destination_name, is_overlayfs_whiteout, options->process_passthrough_whiteouts);

```

Reply:
oh wait... :) 
```
  /* This strange code structure is to preserve testing
   * coverage of both `ostree_repo_checkout_tree` and
   * `ostree_repo_checkout_at` until such time as we have a more
   * convenient infrastructure for testing C APIs with data.
   */
  if (opt_disable_cache || opt_whiteouts || opt_require_hardlinks ||
      opt_union_add || opt_force_copy || opt_force_copy_zerosized ||
      opt_bareuseronly_dirs || opt_union_identical ||
      opt_skiplist_file || opt_selinux_policy || opt_selinux_prefix ||
      opt_process_passthrough_whiteouts)
    {
```